cypress/e2e/notes-100/comments-on-a-note.cy.js

```diff
@@ -0,0 +1,29 @@
+import '@percy/cypress'
+import {
+  homepageSetup,
+  returningUserVisitsHomepageWaitForModel,
+  auth0Login,
+} from '../../support/utils'
+
+/** {@link https://github.com/orgs/bldrs-ai/projects/50/views/1?pane=issue&itemId=52262146} */
```

Comment:
Link to issue directly instead of view in project

Reply:
done